### DISCUSSION TITLE
Currently playing audio at bottom of tablist

### DISCUSSION THREAD

**Discussion by JSGRD22** [zen-browser/desktop#1569] [Ideas]
Feature from Arc browser. When media is playing, pin an element to the bottom of the tablist with play/pause/volume etc buttons on all workspaces.

**Comment by TinBlock50** [comment]
I would also be very happy about the possibility of controlling the music player in background taps. I have quickly made two examples of what this could look like in Zen. One under the tabs and one next to the URL bar.
![Zen_Audio_Mockup_Bottom](https://github.com/user-attachments/assets/05c27d62-f831-4d8f-bafd-296950c5d5c7)
![Zen_Audio_Mockup_Top](https://github.com/user-attachments/assets/c843c096-e766-4ef8-8b8e-906854d54c40)

**Reply by JSGRD22** [reply]
First image is exactly what I mean! This feature would be very handy for controlling audio without searching for the tab or workspace playing it.

**Reply by RicardoCA** [reply]
I need this feature too

**Reply by hoangdinhini** [reply]
Hopefully this would be cooked soon.

**Reply by mr-cheffy** [reply]
Update: https://github.com/zen-browser/desktop/pull/6321
